libsrc/utils/ImageResampler.cpp

```diff
@@ -150,3 +271,28 @@ void ImageResampler::yuv2rgb(uint8_t y, uint8_t u, uint8_t v, uint8_t &r, uint8_
 	g = clamp((298 * c - 100 * d - 208 * e + 128) >> 8);
 	b = clamp((298 * c + 516 * d + 128) >> 8);
 }
+
+
+void ImageResampler::yuv2rgbHDR2SDR(uint8_t y, uint8_t u, uint8_t v, uint8_t &r, uint8_t &g, uint8_t &b, unsigned char *lutBuffer)
```

Comment:
There is another yuv conversion function in this class, I assume you just followed it but in the master I changed it. There is already a color conversion utility file called ColorSys and it makes more sense to keep all conversion code there. Can you move this one to ColorSys as well ? Clamp is already there.

Reply:
I revert back my previous attempt to merge as I saw them. OK, I'll move that part of code to ColorSys to unify changes.